docs/build-parachains-rococo.md

```diff
@@ -0,0 +1,428 @@
+2020-08-25 13:32:15 📦 Highest known block at #0
+2020-08-25 13:32:15 〽️ Prometheus server started at 127.0.0.1:9616
+2020-08-25 13:32:15 📪 Libp2p listener () closed: Address already in use (os error 48)
+2020-08-25 13:32:15 📪 Libp2p listener () closed: Address already in use (os error 48)
+2020-08-25 13:32:18 ✨ [Relaychain] Imported #268 (0xb6ba…65e0)
+2020-08-25 13:32:18 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:20 💤 [Relaychain] Idle (3 peers), best: #268 (0xb6ba…65e0), finalized #265 (0x081c…1b35), ⬇ 18.1kiB/s ⬆ 2.4kiB/s
+2020-08-25 13:32:23 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:24 ✨ [Relaychain] Imported #269 (0x2dd2…9964)
+2020-08-25 13:32:25 💤 [Relaychain] Idle (3 peers), best: #269 (0x2dd2…9964), finalized #266 (0xf093…fd8f), ⬇ 1.0kiB/s ⬆ 0.6kiB/s
+2020-08-25 13:32:28 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:30 ✨ [Relaychain] Imported #270 (0xe7f2…3743)
+2020-08-25 13:32:30 💤 [Relaychain] Idle (3 peers), best: #270 (0xe7f2…3743), finalized #267 (0x8bfc…8abb), ⬇ 1.2kiB/s ⬆ 0.5kiB/s
+2020-08-25 13:32:33 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:35 💤 [Relaychain] Idle (3 peers), best: #270 (0xe7f2…3743), finalized #267 (0x8bfc…8abb), ⬇ 0.7kiB/s ⬆ 0.3kiB/s
+2020-08-25 13:32:36 ✨ [Relaychain] Imported #271 (0x256f…8849)
+2020-08-25 13:32:38 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:40 💤 [Relaychain] Idle (3 peers), best: #271 (0x256f…8849), finalized #269 (0x2dd2…9964), ⬇ 1.5kiB/s ⬆ 0.5kiB/s
+2020-08-25 13:32:42 ✨ [Relaychain] Imported #272 (0x709a…c025)
+2020-08-25 13:32:43 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:45 💤 [Relaychain] Idle (3 peers), best: #272 (0x709a…c025), finalized #269 (0x2dd2…9964), ⬇ 0.6kiB/s ⬆ 0.3kiB/s
+```
+
+Note a few things about this new output:
+
+- `Parachain id: Id(200)` is referring to the id of the parachain that we set it to.
+- `Parachain Account: 5Ec4AhPTL6nWnUnw58QzjJvFd3QATwHA3UJnvSD4GVSQ7Gop` is important for cross chain
+  transfers.
+
+## Registering the Parachain on Polkadot-JS Apps
+
+At this point, the parachain is ready to be registered officially in the UI. The first step is to
+head underneath `Developer` and click on the `Sudo` button.
+
+![rococo registration](assets/rococo/rococo-register-parachain.png)
+
+We will need the genesis state from before that we copied and placed in the `initial_head_data`
+section. On this page, you will see other details like the `id`, `ParaInfo`, and `code`. In the `id`
+field, we will want to place the ID of the parachain, in this case it would be `200`. The `ParaInfo`
+should be set to always, and the `code` field should contain the WASM file that was created while
```

Comment:
```suggestion
should be set to `always`, and the `code` field should contain the Wasm file that was created while
```

Reply:
Proper capitalization is "Wasm" - see https://en.wikipedia.org/wiki/WebAssembly

Reply:
Hm, in the [style guide here](https://www.notion.so/web3foundation/W3F-Writing-Style-Guide-6469d6cb5e644bc2b0663cf1f016424c), it says to refer to wasm as WASM. Does this mean that the style guide should also be updated and other parts of the wiki that reference this?